x-pack/plugin/data-frame/src/main/java/org/elasticsearch/xpack/dataframe/action/TransportStopDataFrameTransformAction.java

```diff
@@ -76,9 +76,9 @@ protected void doExecute(Task task, StopDataFrameTransformAction.Request request
             }
 
             dataFrameTransformsConfigManager.expandTransformIds(request.getId(), new PageParams(0, 10_000), ActionListener.wrap(
-                    expandedIds -> {
-                        request.setExpandedIds(new HashSet<>(expandedIds));
-                        request.setNodes(DataFrameNodes.dataFrameTaskNodes(expandedIds, clusterService.state()));
+                    expandedIdsAndHits -> {
```

Comment:
s/expandedIdsAndHits/hitsAndExpandedIds/
to make variable name indicate the order of tuple elements?